data/data/manifests/openshift/feature-gate.yaml.template

```diff
@@ -0,0 +1,6 @@
+apiVersion: config.openshift.io/v1
```

Comment:
We're vendoring this type, and also importing that directory in `cluster.go`.  It might be about as easy to set up this asset based on the Go type without going through the template system, and that will also get you nice compile-time warnings if an openshift/api vendor bump ever drops v1 support, without you needing to come touch a template string that isn't checked at compile-time.

Reply:
[The Proxy config asset][1] is an example of the "use the Go type" approach, in case that's useful.

[1]: https://github.com/openshift/installer/blob/503e90075a69ea891019ea2e3631806aa935bd0a/pkg/asset/manifests/proxy.go